socorro/siglists/irrelevant_signature_re.txt

```diff
@@ -20,7 +21,12 @@ data@app@org\.mozilla\.f.*-\d\.apk@classes\.dex@0x.*
 .*FNODOBFM
 framework\.odex@0x.*
 gfxPlatform::GetPlatform
-google_breakpad::ExceptionHandler::HandleInvalidParameter.*
+glFlush_ExecThread
+glDrawArrays.*
+google_breakpad::CrashGenerationClient::RequestDumpForException
+google_breakpad::ExceptionHandler::SignalHandler
+google_breakpad::ExceptionHandler::WriteMinidumpWithException
+google_breakpad::ReceivePort::WaitForMessage
```

Comment:
These `google_breakpad*` frames are all the crash reporter. We can skip these with reckless abandon.